src/coreclr/jit/gentree.cpp

```diff
@@ -12093,7 +12093,14 @@ void Compiler::gtDispTree(GenTree*     tree,
                 case NI_System_Runtime_CompilerServices_RuntimeHelpers_IsKnownConstant:
                     printf(" isKnownConst");
                     break;
-
+#if defined(FEATURE_SIMD)
+                case NI_SIMD_UpperRestore:
+                    printf(" simd_upperRestr");
+                    break;
+                case NI_SIMD_UpperSave:
+                    printf(" simd_upperSave");
```

Comment:
nit: We can save a character here and make it a bit more readable
```suggestion
                    printf(" simdUpperSave");
```